sirepo/template/srw.py

```diff
@@ -367,6 +370,36 @@ def _extract_coherent_modes(model, out_info):
     return out_file
 
 
+def _get_recent(data_filename, backup_data_filename):
+    if os.path.getmtime(backup_data_filename) > os.path.getmtime(data_filename):
```

Comment:
These names are long. You can call them "secondary" and "primary". To make the code simpler, simply check the size, not the time, since we know the file continues to grow, and the count lines check overrides the date check.

Reply:
You need to check time because you want the most recent. Current and backup files will contain the same number of points but may be of different file sizes due to number to string in the data file.

Reply:
Thanks.

Reply:
Ok, so check time and use line count rather than file size?

Reply:
I updated the [_best_data_file code](https://github.com/radiasoft/sirepo/pull/4869#discussion_r977871466).